docs/newuser.md

```diff
@@ -1,29 +1,41 @@
 # Flow for new users
-This document will go over the flow for getting access to our various services
+This document will go over the flow for getting access to our various services if you are finding difficulties with the specifics of the tasks outlined in this document please see the more detailed [Developer Docs]()
 
-## Publicly Accessible
+## Office 365 Account
+[Office 365](https://portal.office.com), these are provided by emailing DigitalCapabilitiesPMO@homeoffice.gsi.gov.uk, this is not managed by us. Please note that for Single Sign On (SSO) which is using your Office 365 account to log onto other services pop-ups will need to be enabled on your computer.
```

Comment:
We should probably remove this content and remove it from the git history. Perhaps make it more high-level i.e. SSO is managed as part of an on-boarding process, which will be managed by your project manager / delivery manager to the office 365 tema.

Reply:
+1, we tend to get a lot of confusion about this. So what @jon-shanks said makes it clearer that the ACP team would not be able to help with getting O365 access.